lib/core/src/client/preview/start.js

```diff
@@ -352,6 +355,11 @@ export default function start(render, { decorateStory } = {}) {
       const { default: meta, ...exports } = fileExports;
       const { title: kindName, parameters: params, decorators: decos, component } = meta;
 
+      if (loadedKinds.has(kindName)) {
+        throw new Error(`Duplicate kind: ${kindName} defined in multiple files`);
```

Comment:
I'm good with the error being thrown, but it has to be an error a user can understand.

Please provide instructions to the user on what to do to resolve this.